src/Demos/HSP.lagda

```diff
@@ -1,104 +1,81 @@
 \section{Introduction}
-The \agdaalgebras library \cite{ualib_v2.0.1} is a repository of types and programs
-(theorems and proofs) formalizing the foundations of universal algebra in Martin-Löf
-dependent type theory (\mltt) using the \agda programming language \cite{Norell:2007,agdaref}.
-The library now includes an fairly extensive collection of formal definitions, theorems, and
-proofs that codify, in the formal language of type theory, the analogous definitions,
-theorems, and proofs of classical, set-theory-based universal algebra and equational
-logic. As such, the \agdaalgebras library provides many examples exhibiting the
-expressiveness of inductive and dependent types for representing and reasoning about
-general algebraic and relational structures in a formal language.
-\ifshort\else
-The main advantage of
-formalizing mathematics in type theory using a proof assistant (like \agda) is that
-the software checks the correctness of our proofs by a process known as ``type-checking.''
-\fi
-
-The first major milestone of the \agdaalgebras project is a formal proof of \emph{Birkhoff's
-variety theorem} (also known as the \emph{HSP theorem})~\cite{Birkhoff:1935} in dependent
-type theory.
-To the best of our knowledge, this constitutes the first formalization in \mltt, and the
-first machine-verified proof, of Birkoff's celebrated 1935 result.
-
-Our first proof of the HSP theorem, completed in January of
-2021,\footnote{See the
+The \agdaalgebras library \cite{ualib_v2.0.1} formalizes the foundations of universal algebra
+in intensional Martin-Löf type theory (\mltt) using \agda~{Norell:2007,agdaref}.
+The library includes a wide collection of definitions and verified theorems that appropriately codify
+classical set-theory-based universal algebra and equational
+logic.
+
+The first major milestone of the \agdaalgebras project is a proof of \emph{Birkhoff's
+variety theorem} (also known as the \emph{HSP theorem})~\cite{Birkhoff:1935}.
+To the best of our knowledge, this constitutes the first machine-verified proof of Birkoff's
+celebrated 1935 result.
+
+The proof presented here is the second proof. The first proof%
```

Comment:
I prefer, "Presented here is our second formal proof of the HSP theorem. The first proof suffered..."

Otherwise, nice revision.